src/index.html

```diff
@@ -11,8 +14,64 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Stylesheets with the `.scss` extension cannot be linked directly in HTML. Instead, they need to be compiled to `.css`. Please change the `href` attribute to point to a `.css` file.